### PR TITLE
Added BackEnd Web API-STUFFKART

### PR DESCRIPTION
**Pull request by navi121**

### CODE REVIEW

File: StuffKartProject.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (rizwana64) [inline StuffKartProject.sln]: please add this to gitignore

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StuffKartProject.Controllers
+{
+ 
+  [ApiController]
+  public class AddUserDetailsController : ControllerBase
+  {
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: remove extra space here.

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    public AddUserDetailsController(StuffKartContext context)
+    {
+      _context = context;
+
+    }
+    [Route("AddUser")]
+    [HttpPost]
+    public async Task<ActionResult<UserDetail>> PostUserDetail(UserDetail userDetail)
+    {
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: remove extra line break

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  [ApiController]
+  public class AddUserDetailsController : ControllerBase
+  {
+
+    private readonly StuffKartContext _context;
+
+    public AddUserDetailsController(StuffKartContext context)
+    {
+      _context = context;
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: remove extra line break

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+        }
+        await _context.SaveChangesAsync();
+        return Ok();
+      }
+      catch (Exception ex)
+      {
+        return StatusCode(StatusCodes.Status500InternalServerError, new { ex.Message, Type = ex.GetType().ToString() });
+      }
+
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: remove extra line breaks

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      _context.UserDetails.Add(userDetail);
+      try
+      {
+        if (UserDetailExists(userDetail.MobileNumber))
+        {
+          return BadRequest(Messages.InvalidMobileNumber);
+        }
+        await _context.SaveChangesAsync();
+        return Ok();
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: add a line break before return statement always. also please do formatting for all the files. (ctrl+k,ctrl+D will do the formatting in visual studio)

File: StuffKartProject/Controllers/AdminLoginController.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+          return BadRequest(Messages.InvalidDetails);
+        }
+
+        return StatusCode(200);
+      }
+      catch
+      {
+        return BadRequest(Messages.InvalidDetails);
+      }
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AdminLoginController.cs]: extra line break needs to be removed also need formatting in this file.

File: StuffKartProject/Controllers/AdminLoginController.cs [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+    {
+      try
+      {
+        var loginStatus = _context.Roles.Where(m => m.Admin == loginRequest.Admin && m.Password == loginRequest.Password).FirstOrDefault() != null;
+        if (loginStatus == false)
+        {
+          return BadRequest(Messages.InvalidDetails);
+        }
+
+        return StatusCode(200);
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AdminLoginController.cs]: Do return OK() instead of `return StatusCode(200);`

File: StuffKartProject/Controllers/CartDetailsController.cs [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+            _context = context;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<CartDetail>>> GetCartDetails()
+        {
+            return await _context.CartDetails.ToListAsync();
+        }
+
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CartDetailsController.cs]: unnecessary line breaks needs to be removed.

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 10]
```diff
@@ -0,0 +1,319 @@
...
+  [ApiController]
+  public class DashBoardController : ControllerBase
+  {
+    private readonly StuffKartContext _context;
+
+    IImageService _imageService = null;
+
+    //public DashBoardController(IImageService imageService)
+    //{
+    //    _imageService = imageService;
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: remove commented code if that is not required.

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 11]
```diff
@@ -0,0 +1,319 @@
...
+    //}
+
+    public DashBoardController(StuffKartContext context)
+    {
+      _context = context;
+    }
+    //public class FileUpload
+    //{
+    //  public IFormFile files { get; set; }
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: remove commented code

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 12]
```diff
@@ -0,0 +1,319 @@
...
+      try
+      {
+        await _context.SaveChangesAsync();
+        return addproduct.ProductId;
+      }
+      catch (Exception ex)
+      {
+        return 0;
+      }
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: line break

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 13]
```diff
@@ -0,0 +1,319 @@
...
+    //[HttpPost]
+    //public async Task<ActionResult<DashBoard>> SaveFile([FromForm]FileUpload fileObj)
+    //{
+    //    var value = JsonConvert.SerializeObject(fileObj);
+    //    var oImage = JsonConvert.DeserializeObject<DashBoard>(value);
+    //    if (fileObj.files.Length > 0)
+    //    {
+    //        await using(var ms=new MemoryStream())
+    //        {
+    //            fileObj.files.CopyTo(ms);
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: commented code

**Review by rizwana64 — CHANGES_REQUESTED** [review]

File: StuffKartProject/Controllers/AdminLoginController.cs (hunk 8 above)
Comment (navi121) [reply StuffKartProject/Controllers/AdminLoginController.cs]: Changed to return Ok()

**Comment by navi121** [comment]
I have  been updated all the files and pushed it please check with it

File: StuffKartProject/Controllers/AddUserDetailsController.cs [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+
+    public AddUserDetailsController(StuffKartContext context)
+    {
+      _context = context;
+    }
+
+    [Route("AddUser")]
+    [HttpPost]
+    public async Task<ActionResult<UserDetail>> PostUserDetail(UserDetail userDetail)
+    {
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AddUserDetailsController.cs]: this can be just `public async Task<UserDetail> PostUserDetail(UserDetail userDetail)`

Comment (rizwana64) [reply StuffKartProject/Controllers/AddUserDetailsController.cs]: also the name of class should be `UserDetails`

File: StuffKartProject/Controllers/AdminLoginController.cs [hunk 15]
```diff
@@ -0,0 +1,38 @@
+using System.Linq;
+using Microsoft.AspNetCore.Mvc;
+using StuffKartProject.Constant;
+using StuffKartProject.Models;
+
+namespace StuffKartProject.Controllers
+{
+  [Route("AdminLogin")]
+  [ApiController]
```
Comment (rizwana64) [inline StuffKartProject/Controllers/AdminLoginController.cs]: this controller is not required.
there should be only controller to login all users and you should use a flag to differentiate whether its common user or admin user. 
like isAdmin = true, its admin user else its common user.
Also table should also be same -> user details (username,password,isAdmin(true,false))
also name should be fixed

File: StuffKartProject/Controllers/CartDetailsController.cs [hunk 16]
```diff
@@ -0,0 +1,59 @@
...
+  {
+    private readonly StuffKartContext _context;
+
+    public CartDetailsController(StuffKartContext context)
+    {
+      _context = context;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<CartDetail>>> GetCartDetails()
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CartDetailsController.cs]: `Task<<IEnumerable<CartDetail>>`

File: StuffKartProject/Controllers/AdminLoginController.cs (hunk 15 above)
Comment (navi121) [reply StuffKartProject/Controllers/AdminLoginController.cs]: but how  to store it in table whether we want to have separate column like role(admin/normal)

File: StuffKartProject/Controllers/AddUserDetailsController.cs (hunk 14 above)
Comment (navi121) [reply StuffKartProject/Controllers/AddUserDetailsController.cs]: changed

File: StuffKartProject/Controllers/CartDetailsController.cs [hunk 17]
```diff
@@ -0,0 +1,59 @@
...
+      }
+      catch (DbUpdateException)
+      {
+        return BadRequest(Messages.InvalidContent);
+      }
+      return StatusCode(200);
+    }
+
+    [HttpDelete("{id}")]
+    public async Task<IActionResult> DeleteCartDetail(int id)
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CartDetailsController.cs]: this can be a separate endpoint and it should have a separate route.

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs [hunk 18]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        return StatusCode(200);
+      }
+      catch
+      {
+        return BadRequest(Messages.InvalidDetails);
+      }
+
+    }
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: space

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs [hunk 19]
```diff
@@ -0,0 +1,40 @@
...
+      try
+      {
+        var resetStatus = _context.UserDetails.Where(m => m.Email == request.Email).FirstOrDefault();
+
+        if (resetStatus == null)
+        {
+          return BadRequest(Messages.InvalidDetails);
+        }
+
+        return StatusCode(200);
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: return OK()

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs [hunk 20]
```diff
@@ -0,0 +1,40 @@
...
+      _context = context;
+    }
+
+    [HttpPost]
+    [Route("Forget")]
+    public ActionResult<UserDetail> CheckSecurity([FromBody] UserDetail request)
+    {
+      try
+      {
+        var resetStatus = _context.UserDetails.Where(m => m.Email == request.Email).FirstOrDefault();
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: movie all logical part to a service

Comment (rizwana64) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: and inject the service in controller

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 21]
```diff
@@ -0,0 +1,58 @@
...
+  {
+    private readonly StuffKartContext _context;
+
+    public DashBoardController(StuffKartContext context)
+    {
+      _context = context;
+    }
+
+    [HttpGet]
+    public IEnumerable<DashBoard> GetProductDetails()
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: separate end point/controller for getting product details

Comment (rizwana64) [reply StuffKartProject/Controllers/DashBoardController.cs]: very misleading that route says add product but you are getting product details as well here

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 22]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      return _context.Products.ToList();
+    }
+
+    [HttpPost]
+    public async Task<int> ProductDetail(DashBoard addproduct)
+    {
+      addproduct.Image = "n";
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: why is this required ` addproduct.Image = "n";` ??

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 23]
```diff
@@ -0,0 +1,58 @@
...
+      _context.Products.Add(addproduct);
+      try
+      {
+        await _context.SaveChangesAsync();
+
+        return addproduct.ProductId;
+      }
+      catch (Exception)
+      {
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: exception is not being used, not throwing exception ?

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+        return 0;
+      }
+
+    }
+
+    [HttpGet("{ProductName}")]
+    public async Task<ActionResult<DashBoard>> GetProduct(string ProductName)
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: separate endpoint

File: StuffKartProject/Controllers/FileUploadController.cs [hunk 25]
```diff
@@ -0,0 +1,46 @@
...
+          using (var ms = new MemoryStream())
+          {
+            files.CopyTo(ms);
+            var img = ms.ToArray();
+            string s = Convert.ToBase64String(img);
+            image.Image = s;
+          }
+          _context.SaveChanges();
+
+          return StatusCode(200);
```
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: return OK()

File: StuffKartProject/Controllers/FileUploadController.cs [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    [HttpPost("{id}")]
+    public async Task<ActionResult<DashBoard>> SaveFile(int id, IFormFile files)
+    {
+      var image = _context.Products.Where(m => m.ProductId == id).FirstOrDefault();
+      if (files != null)
+      {
+        if (files.Length > 0)
+        {
```
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: logic goes in a service

File: StuffKartProject/Controllers/CartDetailsController.cs (hunk 16 above)
Comment (navi121) [reply StuffKartProject/Controllers/CartDetailsController.cs]: rectified

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs (hunk 18 above)
Comment (navi121) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: deleted

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs (hunk 19 above)
Comment (navi121) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: changed

File: StuffKartProject/Controllers/FileUploadController.cs (hunk 25 above)
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: return OK()

File: StuffKartProject/Controllers/FileUploadController.cs [hunk 27]
```diff
@@ -0,0 +1,46 @@
...
+            string s = Convert.ToBase64String(img);
+            image.Image = s;
+          }
+          _context.SaveChanges();
+
+          return StatusCode(200);
+        }
+      }
+
+      return BadRequest();
```
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: with a message ?

File: StuffKartProject/Controllers/FileUploadController.cs [hunk 28]
```diff
@@ -0,0 +1,46 @@
...
+    {
+      _context = context;
+    }
+
+    [HttpPost("{id}")]
+    public async Task<ActionResult<DashBoard>> SaveFile(int id, IFormFile files)
+    {
+      var image = _context.Products.Where(m => m.ProductId == id).FirstOrDefault();
+      if (files != null)
+      {
```
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: move logic to a service

File: StuffKartProject/Controllers/FileUploadController.cs [hunk 29]
```diff
@@ -0,0 +1,46 @@
...
+  {
+    private readonly StuffKartContext _context;
+    public FileUploadController(StuffKartContext context)
+    {
+      _context = context;
+    }
+
+    [HttpPost("{id}")]
+    public async Task<ActionResult<DashBoard>> SaveFile(int id, IFormFile files)
+    {
```
Comment (rizwana64) [inline StuffKartProject/Controllers/FileUploadController.cs]: `Task<DashBoard>`

File: StuffKartProject/Controllers/GetUserDetailsController.cs [hunk 30]
```diff
@@ -0,0 +1,31 @@
...
+    {
+      var userDetail = await _context.UserDetails.FindAsync(id);
+      if (userDetail == null)
+      {
+        return NotFound();
+      }
+
+      return userDetail;
+    }
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/GetUserDetailsController.cs]: remove space

File: StuffKartProject/Controllers/SearchBarController.cs [hunk 31]
```diff
@@ -0,0 +1,61 @@
...
+  {
+    private readonly StuffKartContext _context;
+
+    public SearchBarController(StuffKartContext context)
+    {
+      _context = context;
+    }
+
+    [HttpGet]
+    public async Task<ActionResult<IEnumerable<DashBoard>>> GetProducts()
```
Comment (rizwana64) [inline StuffKartProject/Controllers/SearchBarController.cs]: separate end points with proper naming convention

Comment (rizwana64) [reply StuffKartProject/Controllers/SearchBarController.cs]: Task<IEnumerable<DashBoard>>

File: StuffKartProject/Controllers/SearchBarController.cs [hunk 32]
```diff
@@ -0,0 +1,61 @@
...
+    {
+      return await _context.Products.ToListAsync();
+    }
+
+    [HttpGet("{productName}")]
```
Comment (rizwana64) [inline StuffKartProject/Controllers/SearchBarController.cs]: separate endpoint

File: StuffKartProject/Models/StuffKartContext.cs [hunk 33]
```diff
@@ -0,0 +1,187 @@
+using Microsoft.EntityFrameworkCore;
+
+#nullable disable
+
+namespace StuffKartProject.Models
+{
+  public partial class StuffKartContext : DbContext
+  {
+    public StuffKartContext(DbContextOptions<StuffKartContext> options) : base(options)
+    {
```
Comment (rizwana64) [inline StuffKartProject/Models/StuffKartContext.cs]: why is this having a constructor ? options is not being used anywhere else

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs (hunk 20 above)
Comment (navi121) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: created service and injected

File: StuffKartProject/Controllers/DashBoardController.cs (hunk 22 above)
Comment (navi121) [reply StuffKartProject/Controllers/DashBoardController.cs]: table not accepting image as null values even I have given as not null its showing some error so I just pass a value as "n" first and by return product id I am changing that n values as imagestring value

File: StuffKartProject/Controllers/DashBoardController.cs (hunk 23 above)
Comment (navi121) [reply StuffKartProject/Controllers/DashBoardController.cs]: no when there is an error in providing a values it should return 0 so that when we pass productId as 0 there is product in Id =0 so it will return some error message to user like enter valid details

File: StuffKartProject/Controllers/DashBoardController.cs (hunk 21 above)
Comment (navi121) [reply StuffKartProject/Controllers/DashBoardController.cs]: created a separate controller for getting product details

File: StuffKartProject/Controllers/DashBoardController.cs (hunk 24 above)
Comment (navi121) [reply StuffKartProject/Controllers/DashBoardController.cs]: sorry this one I have not used anywhere , I have just made it to check whether its working. I have deleted now.

File: StuffKartProject/Controllers/FileUploadController.cs (hunk 25 above)
Comment (navi121) [reply StuffKartProject/Controllers/FileUploadController.cs]: changed

File: StuffKartProject/Controllers/FileUploadController.cs (hunk 29 above)
Comment (navi121) [reply StuffKartProject/Controllers/FileUploadController.cs]: changed

File: StuffKartProject/Controllers/GetUserDetailsController.cs (hunk 30 above)
Comment (navi121) [reply StuffKartProject/Controllers/GetUserDetailsController.cs]: removed it

File: StuffKartProject/Controllers/SearchBarController.cs (hunk 31 above)
Comment (navi121) [reply StuffKartProject/Controllers/SearchBarController.cs]: sorry this too also I didn't used it anywhere ,I just used it for just checking. I will delete it

File: StuffKartProject/Models/StuffKartContext.cs (hunk 33 above)
Comment (navi121) [reply StuffKartProject/Models/StuffKartContext.cs]: yes I have deleted it

File: StuffKartProject/Controllers/SearchBarController.cs (hunk 32 above)
Comment (navi121) [reply StuffKartProject/Controllers/SearchBarController.cs]: done

File: StuffKartProject/Controllers/FileUploadController.cs (hunk 27 above)
Comment (navi121) [reply StuffKartProject/Controllers/FileUploadController.cs]: okay

File: StuffKartProject/Controllers/FileUploadController.cs (hunk 26 above)
Comment (navi121) [reply StuffKartProject/Controllers/FileUploadController.cs]: done it

File: StuffKartProject/Controllers/CartDetailsController.cs [hunk 34]
```diff
@@ -0,0 +1,54 @@
...
+    private readonly ICartDetailsService _cartDetailsService;
+    private readonly ILogger _logger;
+
+    public CartDetailsController(ICartDetailsService cartDetailsService, ILogger<CartDetailsController> logger)
+    {
+      _cartDetailsService = cartDetailsService;
+      _logger = logger;
+    }
+    
+    [HttpPost("{User}")]
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CartDetailsController.cs]: [HttpPost("{user}")] small case user

File: StuffKartProject/Controllers/CategoryController.cs [hunk 35]
```diff
@@ -0,0 +1,43 @@
...
+  [ApiController]
+  public class CategoryController : ControllerBase
+  {
+    private readonly ICategoryService _categoryService;
+    private readonly ILogger _logger;
+    public CategoryController(ICategoryService categoryService, ILogger<CategoryController> logger)
+    {
+      _categoryService = categoryService;
+      _logger = logger;
+    }
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CategoryController.cs]: line break needed after constructor

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs [hunk 36]
```diff
@@ -0,0 +1,52 @@
...
+using StuffKartProject.Constant;
+using StuffKartProject.Models;
+using StuffKartProject.Services.Interfaces;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StuffKartProject.Controllers
+{
+  [Route("ForgetPassword")]
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: route should be ForgotPassword.
name of the controller and action method should be ForgotPasswordController and ResetPassWordController respectively.
Naming does not seem appropriate in lots of places, take a look around.

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs [hunk 37]
```diff
@@ -0,0 +1,52 @@
...
+
+        return Ok();
+      }
+      catch (Exception ex)
+      {
+        _logger.LogError("Received Exception Error while running");
+
+        return StatusCode(StatusCodes.Status500InternalServerError, new { ex.Message, Type = ex.GetType().ToString() });
+      }
+
```
Comment (rizwana64) [inline StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: remove this line break

File: StuffKartProject/Controllers/DashBoardController.cs (hunk 23 above)
Comment (rizwana64) [reply StuffKartProject/Controllers/DashBoardController.cs]: if you are returning 0 how would you ever know what is the exception message ?
this is wrong , if there is an error in the request then it should be returned as 400 bad request or 422 Unprocessable Entity and that should be captured in the beginning itself, it should not be in a catch.

File: StuffKartProject/Controllers/DashBoardController.cs [hunk 38]
```diff
@@ -0,0 +1,44 @@
...
+        _logger.LogInformation("Receiving Products Succesfully and Added to Table");
+
+        return productId;
+      }
+      catch (Exception ex)
+      {
+        _logger.LogWarning("Received Exception Error while running");
+
+        return 0;
+        //return StatusCode(StatusCodes.Status500InternalServerError, new { ex.Message, Type = ex.GetType().ToString() });
```
Comment (rizwana64) [inline StuffKartProject/Controllers/DashBoardController.cs]: remove commented code from all places

File: StuffKartProject/Controllers/CartDetailsController.cs (hunk 34 above)
Comment (navi121) [reply StuffKartProject/Controllers/CartDetailsController.cs]: I have changed

File: StuffKartProject/Controllers/CategoryController.cs (hunk 35 above)
Comment (navi121) [reply StuffKartProject/Controllers/CategoryController.cs]: added line break after constructor

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs (hunk 36 above)
Comment (navi121) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: Changed controller  name to ForgetPasswordController

File: StuffKartProject/Controllers/CheckSecurityAnswerController.cs (hunk 37 above)
Comment (navi121) [reply StuffKartProject/Controllers/CheckSecurityAnswerController.cs]: removed line break
